frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Add the relevant parts from this to the root README.md - e.g. how to run the app (see what was done for the backend section)